cordapp-upgrades/contracts/v1-contract/build.gradle

```diff
@@ -0,0 +1,75 @@
+apply plugin: 'kotlin'
+apply plugin: 'net.corda.plugins.cordapp'
+apply plugin: 'net.corda.plugins.cordformation'
+apply plugin: 'net.corda.plugins.quasar-utils'
+
+jar {
+    baseName = "obligation"
+    archiveName = "obligation-contracts.jar"
+    exclude '**/log4j2*.xml'
+}
+
+cordapp {
+    targetPlatformVersion 3
+    minimumPlatformVersion 3
+    contract {
+        name "Obligation CorDapp"
+        vendor "Corda Open Source"
+        licence "Apache License, Version 2.0"
+        versionId 1
+    }
+    version = 1
```

Comment:
Is this needed?

Reply:
I think what this does is change the archive name to append a 1 to it - except that behaviour is then overridden by directly setting the archive name in the jar {} block. So I think this can go.